lab-michael/lib/fs-readfile.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+
+module.exports = function(cb) {
+  var textArray = [];
+  fs.readFile(__dirname + '/../data/one.txt', function(err, data) {
+    if (err) {
```

Comment:
You can clean these error handling lines up a bit and put them each on one line, like so: `if (err) console.error(err)`. Note that I used `console.error()`, which should be used when logging errors, and more so in production you should actually `throw err` to actually instantiate the error rather than just logging it.